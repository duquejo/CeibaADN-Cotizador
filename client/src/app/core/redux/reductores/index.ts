import { combineReducers } from 'redux';

import calendarios from './calendarios/calendariosReductor';
import categoriasUsuarios from './categoriasusuarios/categoriaUsuariosReductor';
import centrosVacacionales from './centrosvacacionales/centrosVacacionalesReductor';
import ui from './ui/modalReductor';

export default combineReducers({
    calendarios, 
    categoriasUsuarios,
    centrosVacacionales,
    ui
});
