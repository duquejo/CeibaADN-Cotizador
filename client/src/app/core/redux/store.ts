import { applyMiddleware, compose, createStore } from 'redux';
import rootReducer from './reductores';
import thunk from 'redux-thunk';

declare const window: any;

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const store = createStore(
  rootReducer,
  composeEnhancers(applyMiddleware(thunk))
);

export default store;
