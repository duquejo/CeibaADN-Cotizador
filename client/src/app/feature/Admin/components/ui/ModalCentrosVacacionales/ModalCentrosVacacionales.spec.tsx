import * as React from 'react';
import { RenderResult, render } from '@testing-library/react';
import { ModalCentrosVacacionales } from '.';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import rootReducer from '../../../../../core/redux/reductores';

describe('ModalCentrosVacacionales test', () => {

  let componentWrapper: RenderResult;

  beforeEach(() => {
    const store = createStore( rootReducer );
    componentWrapper = render(
        <Provider store={ store }>
            <ModalCentrosVacacionales/>
        </Provider> 
    );
  });

  it('DeberÃ­a coincidir con el snapshot', () => {
    expect( componentWrapper.container ).toMatchSnapshot();
  });
  
});
