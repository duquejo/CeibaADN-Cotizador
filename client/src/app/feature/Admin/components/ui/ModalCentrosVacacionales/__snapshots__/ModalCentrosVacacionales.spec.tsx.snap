// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalCentrosVacacionales test Deber√≠a coincidir con el snapshot 1`] = `<div />`;
